server/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const knex = require('../database/db');
+const axios = require('axios');
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+
+// load initial/refresh feed
+app.get('/users/:user_id/initial_feed', (req, res) => {
+  const userId = req.params.user_id;
+
+  Promise.all([
+    axios.get(`{get elenas ip address}/users/${userId}/post_feed/0`),
+    axios.get(`{get chadams ip address}/users/${userId}/ad_feed/0`),
+  ])
+    .then((response) => {
+      const { feed } = response[0];
+      const { ad } = response[1];
+      const nextPostIndex = response[0].next_post_index;
+      const nextAdIndex = response[1].next_ad_index;
+
+      // attach ad to feed
+      feed.push(ad);
+
+      // store indices in db
+      knex('feed_indices')
+        .where('user_id', '=', userId)
```

Comment:
Small nit - should be able to use ".where('user_id', userId)" like you did below